### PR TITLE
Add is_trial_period field

### LINKED ISSUE #61: is_trial_period and is_in_intro_offer_period attributes missing from InAppReceipt
The items in the `'receipt' => 'in_app' => []` and `'latest_receipt_info' => []'` have two fields on them that are currently ignored by the InAppReceipt class that turns them into objects. I wanted to check if there's a reason they're currently being ignored - if not I'm happy to submit a pull request to parse and store those values.

### PR DESCRIPTION
**Pull request by rlho**
Fixes #61
In the issue, @drewciminico says that he is preparing to make a pull request on this issue. 
However, that pull request has not been submitted for more than 10 days. I would like to use 'is_trial_period' fields reported in the issue immediately, so I made a pull request instead of him.

### CODE REVIEW

File: lib/venice/in_app_receipt.rb [hunk 1]
```diff
@@ -54,6 +54,7 @@ def initialize(attributes = {})
       @purchased_at = DateTime.parse(attributes['purchase_date']) if attributes['purchase_date']
       @app_item_id = attributes['app_item_id']
       @version_external_identifier = attributes['version_external_identifier']
+      @is_traial_period = to_bool(attributes['is_trial_period'])
```
Comment (luizkowalski) [inline lib/venice/in_app_receipt.rb]: can you fix this typo?

File: lib/venice/in_app_receipt.rb [hunk 2]
```diff
@@ -101,5 +103,10 @@ def to_json
     def is_number?(string)
       !!(string && string.to_s =~ /^[0-9]+$/)
     end
+
+    def to_bool(string)
```
[the remark this answers is not in the document]
Comment (rlho) [reply lib/venice/in_app_receipt.rb]: Sorry for late reply.
When you get the receipt  from Apple,  is_trial_period column is not boolean but string.
For example `is_trial_period: 'false'`.
I think it's better to deal this column in boolean, so I define this method.

File: lib/venice/in_app_receipt.rb (hunk 1 above)
Comment (rlho) [reply lib/venice/in_app_receipt.rb]: Sorry for late reply. I fixed it.

File: lib/venice/in_app_receipt.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (dankimio) [reply lib/venice/in_app_receipt.rb]: Can we just write `string.to_s == 'true'`?

**Comment by Danila** [comment]
@dankimio hey! 
Is there something missing here? Can we merge this?

**Pull request merged**